includes/debug-info.php

```diff
@@ -144,10 +144,10 @@ function add_debug_info( $info ) {
 /**
  * Get and format internal connections.
  *
- * @deprecated x.x.x Use get_formatted_internal_connections
+ * @deprecated 2.0.4 Use get_formatted_internal_connections
```

Comment:
I assume this was actually the last release and not 2.0.5 (same with other since updates in this file)?

Reply:
Yes, these changes were done in the 2.0.4 release but these updates were missed